.github/workflows/publish-next.yml

```diff
@@ -2,14 +2,23 @@ name: Publish next
 
 on:
   workflow_dispatch:
-  schedule:
-    - cron: '0 */12 * * *'
+  # Temporarily off while we work on publishing automatic full releases
+  # schedule:
+  #   - cron: '0 */12 * * *'
+  push:
+    branches:
+      - main
 
 jobs:
   publish:
-    name: Publish to npm
+    name: Publish @next release to npm
     runs-on: ubuntu-latest
     steps:
+      - name: Cancel previous publish attempts
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ github.token }}
```

Comment:
Y'know... seeing this here is really kinda ironic after you said just before .. well this 

![image](https://user-images.githubusercontent.com/4019718/153663082-e2470c45-1539-49b0-bcf7-5508dfd2c16c.png)

IMHO you shouldn't have this here because there are perfectly valid reasons for publishing both `0.x.y-asd822a` and `0.x.yrty32a` even though those 2 commits were added right after one another, the second while CI of the first was still running.

Reply:
More concerned over ratelimits / deprecate cycles clashing, so I'd say it's fine to have it now